### PR TITLE
test(e2e): add e2e and visual tests for Mission Status (plus a11y)

### LINKED ISSUE #7461: Add e2e and visual tests for Mission Status
#### Summary

We should add e2e and visual tests to cover the basic functionalities of Mission Status

### PR DESCRIPTION
**Pull request by ozyx**
Closes #7461 

### Describe your changes:

- Add e2e and visual test suites for Mission Status
- Add a11y wherever needed
- Enable the ExampleUser to set Mission Status so we can test the feature

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Has this been smoke tested?
* [x] Have you associated this PR with a `type:` label? Note: this is not necessarily the same as the original issue.
* [x] Have you associated a milestone with this PR? Note: leave blank if unsure.
* [ ] Is this a breaking change to be called out in the release notes?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?

### CODE REVIEW

**Review by scottbell — APPROVED** [review]
Nice tests!

File: e2e/tests/visual-a11y/missionStatus.visual.spec.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+import percySnapshot from '@percy/playwright';
+import { fileURLToPath } from 'url';
+
+import { expect, test } from '../../pluginFixtures.js';
```
Comment (unlikelyzero) [inline e2e/tests/visual-a11y/missionStatus.visual.spec.js:25]: try to import from avpFixtures.js

File: src/api/overlays/components/DialogComponent.vue [hunk 2]
```diff
@@ -25,16 +25,16 @@
     <div class="c-message__icon" :class="['u-icon-bg-color-' + iconClass]"></div>
     <div class="c-message__text">
       <!-- Uses flex-column -->
-      <div v-if="title" class="c-message__title">
+      <div v-if="title" class="c-message__title" aria-label="Dialog title">
```
Comment (unlikelyzero) [inline src/api/overlays/components/DialogComponent.vue:28]: so i think in this case, we shouldn't be using an aria label if it's not actually interactive

**Review by unlikelyzero — APPROVED** [review]
I'm pretty sure the aria-label isn't necessary as I noted. It would be better to use HTML5

File: src/api/overlays/components/DialogComponent.vue (hunk 2 above)
Comment (ozyx) [reply src/api/overlays/components/DialogComponent.vue:28]: I guess ideally these would be actual header and text elements for the implicit roles. Will put a fixme in here

**Pull request merged**